bot/cogs/chess/__init__.py

```diff
@@ -0,0 +1,11 @@
+import logging
+
+from bot.bot import Friendo
+from ._cog import Chess
+
+logger = logging.getLogger("chess")
+
+
+async def setup(bot: Friendo) -> None:
```

Comment:
this doesn't have to be async since bot.add_cog also has a synchronous counterpart